src/main/java/org/humancellatlas/ingest/submission/SubmissionEnvelope.java

```diff
@@ -111,8 +111,11 @@ private static Logger getLog() {
         List<SubmissionGraphValidationState> allowedStates = new ArrayList<>();
         switch (fromState) {
             case PENDING:
-                allowedStates.add(SubmissionGraphValidationState.VALIDATING);
+                allowedStates.add(SubmissionGraphValidationState.REQUESTED);
                 break;
+            case REQUESTED:
+                allowedStates.add(SubmissionGraphValidationState.PENDING);
```

Comment:
I am not sure about this transition.

Reply:
Why not? It's neccesary in case graph validation fails for some reason. I'm talking about it going invalid, I mean actually an error with the process

Reply:
@ke4 i think it's like a "Queued" state

Reply:
Yeah maybe queued is a better name...